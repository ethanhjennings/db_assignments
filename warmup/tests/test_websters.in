hello
HELLO
HellO
apple
1 t
a
zythepsary
doohicky
whoosawhatsit
aselfsijesdlkfjsldfsueibvslkeifsleirhjslivmsliefhslkmlsdijfoaslifselivnslieflsiefmskleihflseinksueh
.quit
