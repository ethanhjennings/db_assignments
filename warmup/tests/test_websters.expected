Loading 'websters_dict.json'...
Loaded 102413 words.
Enter a word to lookup:
---------------------
HELLO:
Hel*lo", interj. & n.

Defn: See Halloo.
---------------------
Enter a word to lookup:
---------------------
HELLO:
Hel*lo", interj. & n.

Defn: See Halloo.
---------------------
Enter a word to lookup:
---------------------
HELLO:
Hel*lo", interj. & n.

Defn: See Halloo.
---------------------
Enter a word to lookup:
----------------------------------------------------------------------
APPLE:
Ap"ple, n. Etym: [OE. appel, eppel, AS. æppel, æpl; akin to Fries. &
D. appel, OHG, aphul, aphol, G. apfel, Icel. epli, Sw. äple, Dan.
æble, Gael. ubhall, W. afal, Arm. aval, Lith. ob, Russ. iabloko; of
unknown origin.]

1. The fleshy pome or fruit of a rosaceous tree (Pyrus malus)
cultivated in numberless varieties in the temperate zones.

Note: The European crab apple is supposed to be the original kind,
from which all others have sprung.

2. (bot.)

Defn: Any tree genus Pyrus which has the stalk sunken into the base
of the fruit; an apple tree.

3. Any fruit or other vegetable production resembling, or supposed to
resemble, the apple; as, apple of love, or love apple (a tomato),
balsam apple, egg apple, oak apple.

4. Anything round like an apple; as, an apple of gold.

Note: Apple is used either adjectively or in combination; as, apple
paper or apple-paper, apple-shaped, apple blossom, apple dumpling,
apple pudding. Apple blight, an aphid which injures apple trees. See
Blight, n.
 -- Apple borer (Zoöl.), a coleopterous insect (Saperda candida or
bivittata), the larva of which bores into the trunk of the apple tree
and pear tree.
 -- Apple brandy, brandy made from apples.
 -- Apple butter, a sauce made of apples stewed down in cider.
Bartlett.
 -- Apple corer, an instrument for removing the cores from apples.
 -- Apple fly (Zoöl.), any dipterous insect, the larva of which
burrows in apples. Apple flies belong to the genera Drosophila and
Trypeta.
 -- Apple midge (Zoöl.) a small dipterous insect (Sciara mali), the
larva of which bores in apples.
 -- Apple of the eye, the pupil.
 -- Apple of discord, a subject of contention and envy, so called
from the mythological golden apple, inscribed "For the fairest,"
which was thrown into an assembly of the gods by Eris, the goddess of
discord. It was contended for by Juno, Minerva, and Venus, and was
adjudged to the latter.
 -- Apple of love, or Love apple, the tomato (Lycopersicum
esculentum).
 -- Apple of Peru, a large coarse herb (Nicandra physaloides) bearing
pale blue flowers, and a bladderlike fruit inclosing a dry berry.
 -- Apples of Sodom, a fruit described by ancient writers as
externally of air appearance but dissolving into smoke and ashes
plucked; Dead Sea apples. The name is often given to the fruit of
Solanum Sodomæum, a prickly shrub with fruit not unlike a small
yellow tomato.
 -- Apple sauce, stewed apples. [U. S.] -- Apple snail or Apple shell
(Zoöl.), a fresh-water, operculated, spiral shell of the genus
Ampullaria.
 -- Apple tart, a tart containing apples.
 -- Apple tree, a tree naturally bears apples. See Apple, 2.
 -- Apple wine, cider.
 -- Apple worm (Zoöl.), the larva of a small moth (Carpocapsa
pomonella) which burrows in the interior of apples. See Codling moth.
 -- Dead Sea Apple. (a) pl. Apples of Sodom. Also Fig. "To seek the
Dead Sea apples of politics." S. B. Griffin. (b) A kind of gallnut
coming from Arabia. See Gallnut.Ap"ple, v. i.

Defn: To grow like an apple; to bear apples. Holland.
----------------------------------------------------------------------
Enter a word to lookup:
---------------------------------------------------------------------
1 T:
2. To enroll; to enter in a list.
Such follow him as shall be registered. Milton.
Registered letter, a letter, the address of which is, on payment of a
special fee, registered in the post office and the transmission and
delivery of which are attended to with particular care.
---------------------------------------------------------------------
Enter a word to lookup:
--------------------------------------------------------------------------------
A:
A (# emph. #).

1. Etym: [Shortened form of an. AS. an one. See One.]

Defn: An adjective, commonly called the indefinite article, and
signifying one or any, but less emphatically.

Defn: "At a birth"; "In a word"; "At a blow". Shak.

Note: It is placed before nouns of the singular number denoting an
individual object, or a quality individualized, before collective
nouns, and also before plural nouns when the adjective few or the
phrase great many or good many is interposed; as, a dog, a house, a
man; a color; a sweetness; a hundred, a fleet, a regiment; a few
persons, a great many days. It is used for an, for the sake of
euphony, before words beginning with a consonant sound [for exception
of certain words beginning with h, see An]; as, a table, a woman, a
year, a unit, a eulogy, a ewe, a oneness, such a one, etc. Formally
an was used both before vowels and consonants.

2. Etym: [Originally the preposition a (an, on).]

Defn: In each; to or for each; as, "twenty leagues a day", "a hundred
pounds a year", "a dollar a yard", etc.A, prep. Etym: [Abbreviated form of an (AS. on). See On.]

1. In; on; at; by. [Obs.] "A God's name." "Torn a pieces." "Stand a
tiptoe." "A Sundays" Shak. "Wit that men have now a days." Chaucer.
"Set them a work." Robynson (More's Utopia)

2. In process of; in the act of; into; to; -- used with verbal
substantives in -ing which begin with a consonant. This is a
shortened form of the preposition an (which was used before the vowel
sound); as in a hunting, a building, a begging. "Jacob, when he was a
dying" Heb. xi. 21. "We'll a birding together." " It was a doing."
Shak. "He burst out a laughing." Macaulay. The hyphen may be used to
connect a with the verbal substantive (as, a-hunting, a-building) or
the words may be written separately. This form of expression is now
for the most part obsolete, the a being omitted and the verbal
substantive treated as a participle.A. Etym: [From AS. of off, from. See Of.]

Defn: Of. [Obs.] "The name of John a Gaunt." "What time a day is it "
Shak. "It's six a clock." B. Jonson.Defn: A barbarous corruption of have, of he, and sometimes of it and
of they. "So would I a done" "A brushes his hat." Shak.Defn: An expletive, void of sense, to fill up the meter
A merry heart goes all the day, Your sad tires in a mile-a. Shak.Defn: A, as a prefix to English words, is derived from various
sources. (1) It frequently signifies on or in (from an, a forms of
AS. on), denoting a state, as in afoot, on foot, abed, amiss, asleep,
aground, aloft, away (AS. onweg), and analogically, ablaze, atremble,
etc. (2) AS. of off, from, as in adown (AS. ofdüne off the dun or
hill). (3) AS. a- (Goth. us-, ur-, Ger. er-), usually giving an
intensive force, and sometimes the sense of away, on, back, as in
arise, abide, ago. (4) Old English y- or i- (corrupted from the AS.
inseparable particle ge-, cognate with OHG. ga-, gi-, Goth. ga-),
which, as a prefix, made no essential addition to the meaning, as in
aware. (5) French à (L. ad to), as in abase, achieve. (6) L. a, ab,
abs, from, as in avert. (7) Greek insep. prefix a without, or
privative, not, as in abyss, atheist; akin to E. un-.

Note: Besides these, there are other sources from which the prefix a
takes its origin.Defn: This word is properly an adjective, but is commonly called the
indefinite article. It is used before nouns of the singular number
only, and signifies one, or any, but somewhat less emphatically. In
such expressions as "twice an hour," "once an age," a shilling an
ounce (see 2d A, 2), it has a distributive force, and is equivalent
to each, every.

Note: An is used before a word beginning with a vowel sound; as, an
enemy, an hour. It in also often used before h sounded, when the
accent of the word falls on the second syllable; as, an historian, an
hyena, an heroic deed. Many writers use a before h in such positions.
Anciently an was used before consonants as well as vowels.
--------------------------------------------------------------------------------
Enter a word to lookup:
----------------------------
ZYTHEPSARY:
Zy*thep"sa*ry, n. Etym: [Gr.

Defn: A brewery. [R.]
----------------------------
Enter a word to lookup:
'doohicky' is not in this dictionary!
Enter a word to lookup:
'whoosawhatsit' is not in this dictionary!
Enter a word to lookup:
'aselfsijesdlkfjsldfsueibvslkeifsleirhjslivmsliefhslkmlsdijfoaslifselivnslieflsiefmskleihflseinksueh' is not in this dictionary!
Enter a word to lookup:
